### PR TITLE
2.13.X DDF-3915 Updated Automatically Starting Solr as a Process Documentation

### PR DESCRIPTION
**Pull request by mcalcote**
#### What does this PR do?
Revise steps in Automatically Starting Solr to account for the file structure in the distribution.

#### Who is reviewing it? 
@ricklarsen @rzwiefel @ahoffer @austinsteffes @garrettfreibott @JGatley @kcover 

#### How should this be tested?
Read through the changes.

#### Any background context you want to provide?
These steps were updated by following them to test automatic starting of the solr process.

#### Screenshots
image

image

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by mcalcote** [comment]
build now

File: distribution/docs/src/main/resources/content/_running/starting-intro.adoc [hunk 1]
```diff
@@ -287,4 +292,10 @@ start -p 8994
...
 +
 Then click `Next >`.
-. Review the settings and click `Finish`. Additional settings can be made by right clicking the task and updating the Properties. It may be necessary to update the Security Options under the task Properties to `Run with highest privileges`.
\ No newline at end of file
+. Review the settings and click `Finish`. Additional settings can be made by right clicking the task and updating the Properties. It may be necessary to update the Security Options under the task Properties to `Run with highest privileges` or even setting user to "SYSTEM".
+Additionally, the process can be set to restart if it fails under the Properties > Settings tab. Depending on the system it may also make sense to delay the process from starting for a few minutes until the machine has fully booted. This can be adjusted under Properties > Triggers > Edit > Advanced Settings > Delay Task
+
+[NOTE]
+====
+If either process is shutdown (${branding} or Solr) the other will continue running.
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/content/_running/starting-intro.adoc]: ❓ I thought I have seen shutdown DDF also shutting down Solr. Am I confused on this statement?

File: distribution/docs/src/main/resources/content/_running/starting-intro.adoc [hunk 2]
```diff
@@ -238,21 +238,26 @@ From the https://lucene.apache.org/solr/guide/7_4/taking-solr-to-production.html
...
+. Prepare the Solr distribution
++
+Move all the contents of the solr directory into a new folder named solr-_version_ (where _version_ is `7.4.0` or whichever release is being used) and compress the folder into a tgz or zip.
 +
 
 . Run the installation script as root:
 +
 -----
-sudo bash ./install_solr_service.sh solr-version.tgz
+sudo bash ./install_solr_service.sh solr-version.tgz -p 8994
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/content/_running/starting-intro.adoc]: ❓ Do we need consider documenting why port `8994`? Can another port be used? If so what are the impacts?

File: distribution/docs/src/main/resources/content/_running/starting-intro.adoc (hunk 1 above)
Comment (mcalcote) [reply distribution/docs/src/main/resources/content/_running/starting-intro.adoc]: That is true under the "normal" use case when the system is configured to manage Solr alongside DDF. This note is in the section where the system is setup as a process that automatically boots. In that case the DDF is not managing Solr and each of them can be stopped independently.

File: distribution/docs/src/main/resources/content/_running/starting-intro.adoc (hunk 2 above)
Comment (mcalcote) [reply distribution/docs/src/main/resources/content/_running/starting-intro.adoc]: That is the just the port we have selected to run Solr on. Another port can be specified in this step and in the previous version I didn't have any specific port listed. I was hoping that the user would be inclined to make their own decision about what port to use and understand that it needed to match the configuration in DDF in the `system.properties` file. I received feedback from several people running through these steps that they used the suggested commands without specifying a port or updating the properties. This meant that Solr was starting on a different port from where DDF was looking for it. 

I updated the command to include the default port in hopes than anyone copying pasting directly from docs without updating their properties will be more likely to have a successful installation.

I will add another note below this step explaining that the system properties should match the port they select to start Solr.

**Comment by mcalcote** [comment]
build now

**Pull request merged**